src/shared/machine-image.h

```diff
@@ -64,7 +64,9 @@ typedef struct Image {
 } Image;
 
 Image *image_unref(Image *i);
-void image_hashmap_free(Hashmap *map);
+static Hashmap* image_hashmap_free(Hashmap *map) {
```

Comment:
Why do you move this to the header file? inline is needed here?

Reply:
Good catch, I added the missing `inline`.

I moved it to the header file because it's used only once, so making it a `static inline` function save some lines of code without much effect on the generated code.